File: C#/Practice/LeetCode/Medium/Partitioning Into Minimum Number Of Deci-Binary Numbers.cs
// https://leetcode.com/problems/partitioning-into-minimum-number-of-deci-binary-numbers/

public class Solution {
    public int MinPartitions(string n) {
        int max = 0;

        for (int i = 0; i < n.Length; i++) {
            int digitValue = (int)Char.GetNumericValue(n[i]);
            
            if (digitValue > max) {
                max = digitValue;
            }
        }

        return max;
    }
}

File: Python/Practice/LeetCode/Medium/Partitioning Into Minimum Number Of Deci-Binary Numbers.py
# https://leetcode.com/problems/partitioning-into-minimum-number-of-deci-binary-numbers/

class Solution:
    def minPartitions(self, n: str) -> int:
        max = 0

        for i in n:
            if max < int(i):
                max = int(i)

        return max